mass_editing/models/mass_object.py

```diff
@@ -20,119 +20,116 @@
 #
 ##############################################################################
```

Comment:
Please change to new OCA header https://github.com/OCA/maintainer-tools/blob/master/template/module/hooks.py#L3